src/main/WMailWindow.js

```diff
@@ -157,6 +160,14 @@ class WMailWindow extends EventEmitter {
   }
 
   /**
+  * Toggles the sidebar
+  */
+  toggleSidebar () {
```

Comment:
@Thomas101 Went out on a limb with this, not sure if it's the correct approach...